src/components/Search/SearchProjects.js

```diff
@@ -8,65 +10,131 @@ import ProjectIcon from './../ProjectIcon'
+const TRENDING_WORDS_QUERY = gql`
+  query getTrendingWords($from: DateTime!, $to: DateTime!) {
+    getTrendingWords(size: 10, from: $from, to: $to, interval: "8h") {
+      datetime
+      topWords {
+        score
+        word
+      }
+    }
+  }
+`
+
+const assetsPredicate = searchTerm => {
+  const upperCaseSearchTerm = searchTerm.toUpperCase()
+  return ({ ticker, name, slug }) =>
+    name.toUpperCase().includes(upperCaseSearchTerm) ||
+    ticker.toUpperCase().includes(upperCaseSearchTerm) ||
+    slug.toUpperCase().includes(upperCaseSearchTerm)
+}
+
+const trendWordsPredicate = searchTerm => {
+  const upperCaseSearchTerm = searchTerm.toUpperCase()
+  return word => word.toUpperCase().includes(upperCaseSearchTerm)
+}
+
 const SearchProjects = ({
-  projectsList,
+  projects,
   isEditingWatchlist,
   isCopyingAssets,
   checkedAssets,
   watchlistItems,
   searchIconPosition,
+  trendWords,
   ...props
 }) => {
-  const projects = projectsList.length > 0 ? projectsList : ALL_PROJECTS
   return (
     <SearchWithSuggestions
-      sorter={({ name: { length: a } }, { name: { length: b } }) => a - b}
       {...props}
       iconPosition={searchIconPosition}
-      data={projects}
-      predicate={searchTerm => {
-        const upperCaseSearchTerm = searchTerm.toUpperCase()
-        return ({ ticker, name, slug }) =>
-          name.toUpperCase().includes(upperCaseSearchTerm) ||
-          ticker.toUpperCase().includes(upperCaseSearchTerm) ||
-          slug.toUpperCase().includes(upperCaseSearchTerm)
-      }}
-      suggestionContent={({ name, ticker, id }) => {
-        const isAssetInList = isEditingWatchlist
-          ? hasAssetById({ listItems: watchlistItems, id })
-          : false
-        return (
-          <div className={styles.projectWrapper}>
-            <div className={styles.projectInfo}>
-              {isCopyingAssets ? (
-                <Checkbox
-                  isActive={checkedAssets.has(id)}
-                  className={styles.checkbox}
-                />
-              ) : (
-                <ProjectIcon className={styles.icon} size={16} name={name} />
-              )}
-              <span className={styles.name}>{name}</span>
-              <span className={styles.ticker}>({ticker})</span>
-            </div>
-            {isEditingWatchlist && (
-              <Icon
-                fill={`var(--${isAssetInList ? 'casper' : 'jungle-green'}`}
-                type={isAssetInList ? 'remove' : 'plus-round'}
-              />
-            )}
-          </div>
-        )
-      }}
+      withMoreSuggestions={false}
+      data={[
+        {
+          predicate: assetsPredicate,
+          title: 'Assets',
+          items: projects,
+          suggestionContent: ({ name, ticker, id }) => {
+            const isAssetInList = isEditingWatchlist
+              ? hasAssetById({ listItems: watchlistItems, id })
+              : false
+            return (
+              <div className={styles.projectWrapper}>
```

Comment:
Maybe new component for suggestionContent?

Reply:
Indeed, better to move into component